api/middleware.go

```diff
@@ -78,21 +78,24 @@ func (a *API) limitEmailSentHandler() middlewareHandler {
 	return func(w http.ResponseWriter, req *http.Request) (context.Context, error) {
 		c := req.Context()
 		config := a.config
-		if config.External.Email.Enabled && !config.Mailer.Autoconfirm {
+		if (config.External.Email.Enabled && !config.Mailer.Autoconfirm) || (config.External.Phone.Enabled) {
```

Comment:
/nit this seems like a little odd place to have `config.External.Phone.Enabled`  since the limiter is named `limitEmailSentHandler`

Maybe we can consider splitting the rate limiters once the fix is deployed.

Reply:
maybe we should also remove this condition: 
```suggestion
		if config.External.Email.Enabled || config.External.Phone.Enabled {
```

Reply:
Fine with me...

Reply:
Let's do this in a follow up PR.